core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="modal-header e2e-test-question-suggestion-review-modal-header">
   <div>
-    <h3 class="oppia-question-header">
+    <h1 class="oppia-question-header">
```

Comment:
Looking at the screenshot, `h1` seems slightly too large. From https://www.w3.org/WAI/tutorials/page-structure/headings/, it seems like the important thing is to make sure the headings are in descending order, so maybe make the header an `h2` and the author header below `h3`. 

@KolliAnitha Would that be fine?